IOMesh 0.11.1 release notes (#146)
iomesh/docs: IOMesh 0.11.1-release notes.md

> ---

Reviewer: 这个 md 直接删掉？

title: IOMesh 0.11.1 Release Notes
author: SmartX Documentation Team 
hide_title: true
id: 0.11.1 Release Notes
sidebar_label: IOMesh 0.11.1 Release Notes
---

# IOMesh 0.11.1 Release Notes

The release notes cover the following topics:

 - [What's New](#new-features)
 - [Component Versions](#component-versions)
 - [Compatibility Notes](#compatibility-notes)

>  - [Download Links](#download-links)

Reviewer: 小标题和之前的发布说明不一致
https://docs.iomesh.com/additional-info/release-notes

## What's New 

### New Feature

> Increase compatibility with Kubernetes versions 1.22 to 1.24.

Reviewer 2: ```suggestion
Add support for Kubernetes versions 1.22 to 1.24.
```

### Improved Features

> - Optimized the method of clearing corresponding LUNs in IOMesh after deleting the PV with its retention policy set to **Retain**. 

Reviewer: clearing --> deleting
retention policy --> reclaim policy

> - Optimized `open-iscsi` containerization so that IOMesh installation and operation will not rely on Linux distros.

Reviewer: Linux distros 用缩写可以吗？官方文档是不是尽量用全称？

Reviewer 2: ```suggestion
- Improved the way of managing `open-iscsi` to ease IOMesh installation on various Linux distros.
```

Author: 想问下软件世界会经常使用 improve the way of 这种表达吗? 单从英语角度不常这么说

Reviewer 2: 我理解应该是一个非常常用的说法呀，https://www.google.com.hk/search?q=%22improve+the+way+of+%22 可以搜索出 75m 条记录，为什么说不常说呢？

> - Simplified Snapshot Controller installation. The Snapshot Controller will be installed with IOMesh. 

Reviewer 2: ```suggestion
- Simplified Snapshot Controller installation. The Snapshot Controller will be installed together with IOMesh. 
```

> ### Fixed Problems

Reviewer 3: 标题下一行是否保留空行呢？上一个标题保留了，这里不一致

Author: 好的

- Fixed the problem that the name of `iscsi-redirectd daemonset` did not conform to the naming convention.

> - Fixed the problem that the operations of PR (Persistent Reservation) might fail due to non-compliance with the SCSI-3 PR protocol.

Reviewer 2: ```suggestion
- Fixed the problem that the operations of SCSI-3 PR (Persistent Reservation) might fail due to non-compliance with the SCSI-3 PR protocol.
```

- Fixed the problem that the time to mount a disk was occasionally too long.
- Fixed the problem that configurations could not take effect when the user customized the NDM image repository.

> - Fixed the problem that the number of PV replicas created or cloned from snapshots did not match that of original PV replicas.

Reviewer: 修复了从快照创建或克隆生成的 PV 的副本数与原 PV 副本数可能不一致的问题。

方式 1: 从快照创建 --> 完整的描述是从 PV 的快照创建
方式 2: 克隆 --> 完整的描述是 克隆 PV

Fixed the problem that the number of replicas of a PV cloned or created from a snapshot might be inconsistent with the number of replicas of the original PV.

> ## Component Versions

Reviewer: 标题和之前不一样
https://docs.iomesh.com/additional-info/release-notes#specifications

| Component | Version|
| -------| -------|
|IOMesh Operator|0.11.1|
| CSI Driver| 2.5.1|
|ZBS|5.1.2|
|Zookeeper|3.5.9|
|Node Disk Manager|1.8.0|
|Hostpath Provisioner|0.5.2|

## Compatibility Notes

### IOMesh and Server Architecture Compatibility  

> IOMesh is compatible with Intel x86_64 and AMD x86_64 architectures.

Reviewer: compatibility 这部分的标题和结构，还有这句话和上次发布说明的描述不一样，是有意调整的吗？

https://docs.iomesh.com/additional-info/release-notes#compatibility

> ### IOMesh and Kubernetes Compatibility

Reviewer 3: 同上，空行问题

> IOMesh is compatible with Kubernetes versions 1.17 to 1.24.

Reviewer: 这里缺少了 openshift

另外为啥不是和上次一样用表格？

### IOMesh and Linux OS Compatibility

> IOMesh is compatible with CentOS 7, CentOS 8, CoreOS, RHEL 7, and RHEL 8.

Reviewer: 这里中文发布说明补充了一句说明，麻烦补充一下～

> 说明：IOMesh 对 Linux 操作系统的版本没有依赖，表格中仅列出经过测试的版本。

https://docs.google.com/document/d/19BGi1U8ZrFYvZRNmrzDcthGdWIlhGW59tKvIxvYG5w8/edit#heading=h.d0rj5en4aweg
